tests/console/validate_zfcp.pm

```diff
@@ -133,10 +134,9 @@ sub verify_scsi_devices {
             validate_script_output("lszfcp -D -b $fcp_channel | grep '$bus_wwpn'", qr/$hctl/);
 
             # Validate scsi devices with specific command for zfcp: lszdev
-            my $names = $lun->{names};
             $bus_wwpn = "$fcp_channel:$wwpn";
-            validate_script_output("lszdev --no-headings zfcp-lun | grep '$names'",
```

Comment:
what do you think about to additionally map all names before the loop and check if the one read belong to that list? So we relax the check asking for the bus instead of the name, but still we check the name is in some set of values.

Reply:
I've explained it in the PR description, we still have a loop over luns and check that have to devices mapped to sda and sdb. So I don't think that we need to add anything.

Reply:
ahm, we still have the device_node_name, you're right, thanks.